docs/_pages/releases.md

```diff
@@ -6,6 +6,32 @@ toc: true
 last_modified_at: 2019-12-30T15:59:57-04:00
 ---
 
+## [Release 0.5.2-incubating](https://github.com/apache/incubator-hudi/releases/tag/release-0.5.2-incubating) ([docs](/docs/0.5.2-quick-start-guide.html))
+
+### Download Information
+ * Source Release : [Apache Hudi(incubating) 0.5.2-incubating Source Release](https://www.apache.org/dist/incubator/hudi/0.5.2-incubating/hudi-0.5.2-incubating.src.tgz) ([asc](https://www.apache.org/dist/incubator/hudi/0.5.2-incubating/hudi-0.5.2-incubating.src.tgz.asc), [sha512](https://www.apache.org/dist/incubator/hudi/0.5.2-incubating/hudi-0.5.2-incubating.src.tgz.sha512))
+ * Apache Hudi (incubating) jars corresponding to this release is available [here](https://repository.apache.org/#nexus-search;quick~hudi)
+
+### Migration Guide for this release
```

Comment:
can we call out the `KeyGenerator` rename? this is critical.. already one user has raised this in slack or mailing list

Reply:
OK, actually IMO, we also need to add it for 0.5.1. Whatever, let's discuss it later.

Reply:
if thats in 0.5.1 [(which I think it is)](https://github.com/apache/incubator-hudi/tree/release-0.5.1/hudi-utilities/src/main/java/org/apache/hudi/utilities/keygen), please add a section to 0.5.1 and not here .. ? 

its fine, we can fix it now atleast..